addons/escoria-core/game/core-scripts/esc/commands/camera_set_zoom_height_block.gd

```diff
@@ -0,0 +1,70 @@
+# `camera_set_zoom_height_block pixels [time]`
+#
+# Zooms the camera in/out so it occupies the given height in pixels.
+# Blocks until the command completes.
+#
+# **Parameters**
+#
+# - *pixels*: Target height in pixels
+# - *time*: Number of seconds the transition should take, with a value of `0`
+#   meaning the zoom should happen instantly (default: `0`)
+#
+# For more details see: https://docs.escoria-framework.org/camera
+#
+# @ESC
+extends ESCBaseCommand
+class_name CameraSetZoomHeightBlockCommand
+
+
+# Tween for blocking
+var _camera_tween: Tween
+
+
+# Return the descriptor of the arguments of this command
+func configure() -> ESCCommandArgumentDescriptor:
+	return ESCCommandArgumentDescriptor.new(
+		1,
+		[TYPE_INT, [TYPE_INT, TYPE_REAL]],
+		[null, 0.0]
+	)
+
+
+# Validate whether the given arguments match the command descriptor
+func validate(arguments: Array):
+	if not .validate(arguments):
+		return false
+
+	if arguments[0] < 0:
+		escoria.logger.error(
+			self,
+			"[%s]: invalid height. Can't zoom to a negative height (%d)."
+					% [get_command_name(), arguments[0]]
+		)
+		return false
```

Comment:
Setting zoom level to 0 for this command results in 
 ESC (2022-11-10T23:06:44) E esc_camera.gd: Tried to set negative or zero zoom level.
So possibly this should be <= 0?